views/partials/search-form-fields.html

```diff
@@ -183,7 +183,7 @@
   <div class="controls">
     <button type="submit"><span>Search</span></button>
     {% if advancedSearchIsVisible %}
```

Comment:
OK, it makes sense now. I was somehow expecting it to show up after setting the filters, but before hitting "Search". Sorry for the noise, I overlooked a few bits of code here :-)